cmd/zar/import/command.go

```diff
@@ -53,6 +54,8 @@ func New(parent charm.Command, f *flag.FlagSet) (charm.Command, error) {
 	f.StringVar(&c.dataPath, "data", "", "location for storing data files (defaults to root)")
 	c.thresh = archive.DefaultLogSizeThreshold
 	f.Var(&c.thresh, "s", "target size of chunk files, as '10MB' or '4GiB', etc.")
+	c.importBufSize = units.Bytes(archive.ImportBufSize)
+	f.Var(&c.importBufSize, "bufsize", "maximum size of data read into memory before flushing to disk '99MB', '4GiB', etc.")
```

Comment:
```suggestion
	f.Var(&c.importBufSize, "bufsize", "maximum size of data read into memory before flushing to disk, as '99MB', '4GiB', etc.")
```
For symmetry with the c.thresh usage string.